### ISSUE TITLE
upgrade to ubuntu 17.10

### ISSUE THREAD

**Issue by zankich** [craigfurman/linux-playground#3]
Hello,

Would you be open to accepting a patch which switches the ubuntu version to 17.10?

I've also noticed that you're updating the [kernel](https://github.com/craigfurman/linux-playground/blob/b60bd72ac49bfd41334864edb585966f0e6b2d60/playbook.yml#L122-L130).

Since 17.10 ships with 4.13, that line would revert the kernel to a previous version, so I'd like to remove that, as well as the [guest additions reinstall](https://github.com/craigfurman/linux-playground/blob/b60bd72ac49bfd41334864edb585966f0e6b2d60/vmup.sh#L51-L65)

**Comment by zankich** [comment]
Actually I would like to bump the kernel to 4.15. Would you also be open to that change?

**Comment by craigfurman** [comment]
Hey @zankich, sorry for not responding earlier, I just discovered my Github notifications have stopped for some reason.

I've bumped to 17.10 in the latest commit, and made a few other changes, including getting rid of the mainline kernel installation. This drops the requirement for the vagrant-vbguest plugin.

The latest 17.10 boxes are on Linux 4.13, is this new enough for your needs, or will you be using 4.14+ features?
